CONTRIBUTING.md

```diff
@@ -1,17 +1,13 @@
-============
-Contributing
-============
+# Contributing
 
 Contributions are welcome, and they are greatly appreciated! Every
-little bit helps, and credit will always be given. 
+little helps, and credit will always be given. 
```

Comment:
Trivial remark: every little _what_ helps? :)

Reply:
Have to agree with Yuriy here. There's something missing now.

Reply:
The automatic spellcheck of my editor gave me this suggestion.

Looking around it seems that one is used more in the UK and the other in the rest of the world. In the end of the day it does not matter much so if you want to change it back feel free to submit a PR.